src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -390,3 +394,50 @@ export const makeDaysToBreakEven = ({
   const daysToBreakEven = BigNumber.max(daysToBreakEvenOrZero, 1).toFixed(0)
   return daysToBreakEven
 }
+
+export const getThorchainFromAddress = async ({
```

Comment:
should probably be a common thorchain util if we have a good spot to put it

Reply:
Agreed to consolidate in a follow-up to avoid stack conflicts